wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -14,7 +14,10 @@ extra-source-files:  README.md
 cabal-version:       >=1.10
 
 library
-  hs-source-dirs:      src
+  hs-source-dirs:      src server
+  ghc-options:         -Wall
+                       -fno-warn-orphans
```

Comment:
:(

I'd rather put this in each file than here. Also can be another PR.

Reply:
i believe that was in the old cabal file, just moved a bit